mmv1/third_party/terraform/services/cloudrun/resource_cloud_run_domain_mapping_test.go

```diff
@@ -109,3 +109,35 @@ resource "google_cloud_run_domain_mapping" "default" {
 }
 `, context)
 }
+
+func TestAccCloudRunDomainMapping_migration(t *testing.T) {
+	acctest.SkipIfVcr(t)
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"namespace":     envvar.GetTestProjectFromEnv(),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	oldVersion := map[string]resource.ExternalProvider{
+		"google": {
+			VersionConstraint: "4.84.0", // a version that doesn't separate user defined labels and system labels
+			Source:            "registry.terraform.io/hashicorp/google",
+		},
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.AccTestPreCheck(t) },
+		CheckDestroy: testAccCheckCloudRunDomainMappingDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config:            testAccCloudRunDomainMapping_cloudRunDomainMappingUpdated1(context),
```

Comment:
What makes this test interesting? It doesn't set any labels.

Reply:
The test function should be `testAccCloudRunDomainMapping_cloudRunDomainMappingUpdated2`